test/performance/odp_bench_buffer.c

```diff
@@ -558,6 +588,563 @@ static int event_flow_id_set(void)
 	return i;
 }
 
+static int buffer_from_event_tm(bench_tm_result_t *res, int repeat_count)
```

Comment:
A matter of taste, but I'd put the basic and tm test variants back-to-back, so it's easier to compare the implementations at a glance.